view/templates/admin/addons.tpl

```diff
@@ -9,12 +9,12 @@
 		<a class="btn" href="{{$baseurl}}/admin/{{$function}}?a=r&amp;t={{$form_security_token}}">{{$reload}}</a>
 		<ul id='pluginslist'>
 		{{foreach $addons as $p}}
-			<li class='plugin {{$p.1}}'>
-				<a class='toggleplugin' href='{{$baseurl}}/admin/{{$function}}/{{$p.0}}?a=t&amp;t={{$form_security_token}}' title="{{if $p.1==on}}Disable{{else}}Enable{{/if}}" ><span class='icon {{$p.1}}'></span></a>
+			<li class="plugin {{$p.1}}">
+				<span class="offset-anchor" id="{{$p.0}}"></span>
```

Comment:
I knew anchors as `<a>` tag, what made you use `<span>` instead? Maybe the hiding CSS below isn't necessary with an `<a>`?